packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -176,7 +177,7 @@ export class WebviewWidget extends BaseWidget implements StatefulWidget {
         this.toDispose.push(this.onMessageEmitter);
 
         this.transparentOverlay = document.createElement('div');
-        this.transparentOverlay.classList.add(MiniBrowserContentStyle.TRANSPARENT_OVERLAY);
+        this.transparentOverlay.classList.add(TRANSPARENT_OVERLAY_STYLE);
```

Comment:
I think you can directly use the `core` class here instead of setting a new variable.

Reply:
I guess it's preference. Personally I prefer to pull magic strings into consts to raise their profile a little :)

Reply:
No problem :)